src/components/SearchBar.vue

```diff
@@ -31,13 +31,34 @@
             },
             onResultSelect(event) {
                 this.map.indoors.setFloor(event.result.data.floor_id);
+
                 this.map.setView(event.result.location.latLng, 20);
+
+                this.route({map: this.map, destination: [event.result.data.lon, event.result.data.lat, event.result.data.floor_id]});
```

Comment:
Given that we are no longer passing in route as prop, this code should not work. Did you run the code before submitting the PR? Please make sure that you do that. I expect the code that is being merged to master has built. Ideally, we would have CI which would check that the code builds before merging to master  but as a developer you should make sure that you run your code before trying to merge it master.

Reply:
Oh, I didn't know we were no longer passing in route as a prop. When I was merging this was a line of code that was already pushed so I decided not to touch it.